test/Mac_test.cpp

```diff
@@ -510,7 +510,7 @@ int MacMessage()
 {
 	HCOSE_MAC hEncObj =
 		COSE_Mac_Init(COSE_INIT_FLAGS_NONE, CBOR_CONTEXT_PARAM_COMMA NULL);
-	char *sz = "This is the content to be used";
+	auto sz = "This is the content to be used";
```

Comment:
I hate auto

Reply:
oh no :D
no problem, lets get rid of it

Reply:
I have problems with JavaScript and python for the same reasons.  I like to be able to read things as typed and be able to make the correct assumptions.  I have done tons of python programming and some JavaScript.  For python since I had to support 2.7 for the last project and it was trying to do unicode I was always getting crashes.

Reply:
I also like to see the type, therefore i only use auto with something clear e.g. we know here the type and with something like templates `auto i = std::make_shared<int>()` as the type is on the right site...